data/src/generate_tsv_summary.py

```diff
@@ -11,15 +11,60 @@
 
 
 def _wiki_name_and_transcription_level(ele: List[str]) -> str:
-    return ele[3] + ele[5]
+    return ele[3] + ele[7]
 
+def _get_modifiers(
+    language: Dict[str, Any], modifier: str
+) -> Dict:
+    try:
+        return language[modifier]
+    except KeyError:
+        return {}
+
+def _handle_script(
```

Comment:
The logic in these `_handle_*` functions is almost identical. Could we not just have a single one?